tests/start-script/BUILD

```diff
@@ -0,0 +1,9 @@
+load(
+    "//:bazel/inline_tests.bzl",
+    "sh_inline_test"
```

Comment:
Do we really need `sh_inline_test` in scope here?

Reply:
We don’t, if you can find another way to make the start script build a Haskell binary without writing the source file to the user’s repository